docs/src/content/docs/ru/guides/sidebar.mdx

```diff
@@ -0,0 +1,524 @@
+
+### Варианты значков
+
+Настройте стиль значка, используя объект с свойствами `text` и `variant`.
+
+`text` представляет содержимое для отображения (например, "New").
+Переопределите стиль `default`, который использует акцентный цвет вашего сайта,
+установив свойство `variant` в одно из следующих значений: `note`, `tip`, `danger`, `caution` или `success`.
+
+```js
+starlight({
+	sidebar: [
+		{
+			label: 'Руководства',
+			items: [
+				// Ссылка с желтым значком "Экспериментально".
+				{
+					label: 'Компоненты',
+					link: '/ru/guides/components/',
+					badge: { text: 'Экспериментально', variant: 'caution' },
+				},
+			],
+		},
+	],
+});
+```
+
+Конфигурация выше создаст следующую боковую панель:
+
+<SidebarPreview
+	config={[
+		{
+			label: 'Руководства',
+			items: [
+				{
+					label: 'Компоненты',
+					link: '/ru/guides/components/',
+					badge: { text: 'Экспериментально', variant: 'caution' },
+				},
+			],
+		},
+	]}
+/>
+
+## Пользовательские HTML-атрибуты
+
+Ссылки также могут включать свойство `attrs` для добавления пользовательских HTML-атрибутов к элементу ссылки.
+
+В следующем примере `attrs` используется для добавления атрибута `target="_blank"`, чтобы ссылка открывалась в новой вкладке,
+а также для применения атрибута `style`, чтобы курсивом выделить метку ссылки:
+
+```js
+starlight({
+	sidebar: [
+		{
+			label: 'Руководства',
+			items: [
+				// Внешняя ссылка на документацию Astro, открывающаяся в новой вкладке.
+				{
+					label: 'Документация Astro',
+					link: 'https://docs.astro.build/',
+					attrs: { target: '_blank', style: 'font-style: italic' },
+				},
+			],
+		},
+	],
+});
+```
+
+Конфигурация выше создаст следующую боковую панель:
+
+<SidebarPreview
+	config={[
+		{
+			label: 'Руководства',
+			items: [
+				{
+					label: 'Документация Astro',
+					link: 'https://docs.astro.build/',
+					attrs: {
+						target: '_blank',
+						style: 'font-style: italic',
+					},
+				},
+			],
+		},
+	]}
+/>
+
+## Интернационализация
+
+Используйте свойство `translations` для ссылок и групп, чтобы перевести текст ссылки или группы на каждый поддерживаемый язык.
+Свойство `label` будет использоваться для языка по умолчанию и для языков без перевода.
+
+```js
+starlight({
+	sidebar: [
+		{
+			label: 'Руководства',
+			translations: {
+				es: 'Guías',
+			},
+			items: [
+				{
+					label: 'Компоненты',
+					translations: {
+						es: 'Componentes',
+					},
+					link: '/guides/components/',
+				},
+				{
+					label: 'Интернационализация (i18n)',
+					translations: {
+						es: 'Internacionalización (i18n)',
+					},
+					link: '/guides/i18n/',
+				},
+			],
+		},
+	],
+});
+```
+
+При просмотре документации на испанском языке будет сгенерирована следующая боковая панель:
+
+<SidebarPreview
+	config={[
+		{
+			label: 'Guías',
+			items: [
+				{ label: 'Componentes', link: '/es/guides/components/' },
+				{ label: 'Internacionalización (i18n)', link: '/es/guides/i18n/' },
+			],
+		},
+	]}
+/>
```

Comment:
I think this may be missing the changes from #881 which adds details about BCP-47 language tags and uses a BCP-47 language tag with a regional subtag for the "Internationalization" section example.

Reply:
done